ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -63,6 +66,18 @@ tests = testGroup "Mempool"
       , testProperty "splitAfterTxSize"                    prop_TxSeq_splitAfterTxSize
       , testProperty "splitAfterTxSizeSpec"                prop_TxSeq_splitAfterTxSizeSpec
       ]
+  , testGroup "MempoolPure"
+      [ testProperty "removed transactions are removed"    prop_pure_removeTxs
+      , testProperty "not removed transactions remain"     prop_pure_removeTxs_contra
+      , testProperty "valid added txs == getTxs"           prop_pure_addTxs_getTxs
+      , testProperty "addTxs txs == mapM addTxs txs"       prop_pure_addTxs_one_vs_multiple
+      , testProperty "snapshotTxs == snapshotTxsAfter zeroIdx"
+                                                           prop_pure_snapshotTxs_snapshotTxsAfter
+      , testProperty "result of addTxs"                    prop_pure_addTxs_result
+      , testProperty "Invalid transactions are never added" prop_pure_InvalidTxsNeverAdded
+      , testProperty "result of getCapacity"               prop_pure_getCapacity
+      , testProperty "Mempool capacity implementation"     prop_pure_Capacity
```

Comment:
```suggestion
      , testProperty "Mempool capacity implementation"     prop_pure_Capacity
```
This description and the property name are vague and actually misleading. Looking at the property, it is more about "tryAddTxs respecting (max) capacity".